kubernetes/model/v1beta1_certificate_signing_request_status.c

```diff
@@ -110,7 +110,7 @@ v1beta1_certificate_signing_request_status_t *v1beta1_certificate_signing_reques
 
 
     v1beta1_certificate_signing_request_status_local_var = v1beta1_certificate_signing_request_status_create (
-        certificate ? certificate->valueint : 0,
+        certificate ? strdup(certificate->valuestring) : NULL,
```

Comment:
in the line 34, 
free v1beta1_certificate_signing_request_status->certificate should be added

Reply:
Good catch! Thank you. I will fix it.